ci/run.sh

```diff
@@ -4,99 +4,50 @@
 
 set -euo pipefail
 
-BUILD_IMAGES=0
+BUILD_IMAGE=0
 PUSH=0
 NOCACHE=
 REPO=shadowsim/shadow-ci
 
-CONTAINERS=(
-    ubuntu:18.04
-    ubuntu:20.04
-    ubuntu:21.10
-    ubuntu:22.04
-    debian:10-slim
-    debian:11-slim
-    fedora:34
-    fedora:35
-    fedora:36
-    quay.io/centos/centos:stream8
-    )
-
-CCS=(
-    gcc
-    clang
-    )
-
-BUILDTYPES=(
-    debug
-    release
-    )
-
-EXTRAS=("ubuntu:18.04;clang;coverage")
-
-NOCACHE=
+CONTAINER=ubuntu:18.04
+CC=gcc
+BUILDTYPE=debug
 
 show_help () {
     cat <<EOF
 Usage: $0 ...
   -h
   -?             Show help
-  -i             Build images
-  -c CONTAINERS  Set containers used in test matrix
-  -C CCS         Set C compilers used in test matrix
-  -b BUILDTYPES  Set build-types used in test matrix
-  -e EXTRAS      Set extra configurations to run
-  -o             Set only configurations to run
+  -i             Build image
+  -c CONTAINER   Set container
+  -C CC          Set C compiler
+  -b BUILDTYPE   Set build-types
   -n             nocache when building Docker images
-  -p             pushes images to dockerhub if set
+  -p             push image to dockerhub
```

Comment:
These changes probably also require https://shadow.github.io/docs/guide/ci.html to be updated.

Reply:
Done